skimage/filters/rank/__init__.py

```diff
@@ -22,6 +22,7 @@
            'subtract_mean',
            'subtract_mean_percentile',
            'median',
+           'median_float',
```

Comment:
Yes, it can but the parameters are different. The idea that emmanuelle shared was to give the output of ndimage.median_filter for float images.